app/helpers/organisations_helper.rb

```diff
@@ -2,4 +2,10 @@ module OrganisationsHelper
   def organisations_options
     Organisation.all.map { |o| [o.title, o.content_id] }
   end
+
+  def primary_publishing_organisation_options(form)
```

Comment:
Maybe we could add this to the other doc types that use an org select? (possibly in another PR)

Reply:
Yes, I was wondering about that. I guess the question is - for the other two types, is the lack of a default a feature? (if the field value isn't usually the current user's org, then the field being empty by default plus the validation for presence acts as a gate, the user can't accidentally save/publish without deliberately putting in the value).

Reply:
(I've also confirmed with Content 2L that this is a desirable feature, so will definitely add it to the other document types in a follow-up PR once this is merged.)